src/main/java/de/tum/in/www1/artemis/domain/Achievement.java

```diff
@@ -37,6 +37,12 @@
     @Column(name = "achievement_type")
     private AchievementType type;
 
+    @Column(name = "parameter")
+    private Long parameter;
+
+    @Column(name = "min_score_to_qualify")
+    private Long minScoreToQualify;
```

Comment:
Also, what does this exactly mean and how is it different to `parameter`?

Reply:
Where the metrics of achievements are not directly point based, the `minScoreToQualify` property can additionally be set.
That means if we take the time based achievements for example and someone hands in an empty submission within a day or simply gets very few to no points that does not deserve an achievement although it would gain an achievement by the time of submission alone. To exclude such cases, we can use this `minScoreToQualify`.